### DISCUSSION TITLE
Welcome to actions-gh-pages-test-repo Discussions!

### DISCUSSION THREAD

**Discussion by rekmarks** [MetaMask/actions-gh-pages-test-repo#96] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tesla809** [comment]
Hello sers.

Is it possible to create Snap that auto-schedules transactions for later on?

**Reply by httpJunkie** [reply]
We currently will kill an idle snap after a certain amount of time. but this is something we could possibly do in the future with permissions. But I'm sure someone could explain it better. 

OFF TOPIC: I'm just digging into Snaps and the docs, for me it's a slightly different mindset as a traditional UI developer, but if you have experience building libraries and npm packages, using tools like GraphQL, REST APIs and JSON/JSON RPC, Snaps is such a cool thing to start hacking on. A lot of the stuff I learned building an open source library have definitely helped me get started with Snaps. This is a great tool for those building Dapps and wanting to get some more custom behavior create new APIs to MetaMask, add new features, or modify existing functionality using internal APIs.

**Comment by jamiels** [comment]
Love to see Metamask and Metamask Snaps introduced to Blockchain NYC!

**Comment by humblesr** [comment]
What is Metamask Snaps?

**Reply by peterduhon** [reply]
Curious about Snaps and the roadmap! 🔥

**Reply by httpJunkie** [reply]
Hello @humblesr,

Have you checked out the MetaMask Snaps docs yet? A few things you can do to get started:

- Install [MetaMask Flask](https://metamask.io/flask/) (experimental MM build devs)
- Flask is needed to install and test your own snaps
- Flask is a parallel distribution of MM with cutting edge features not available in the regular MM extension
- Snaps are an installable plugin for Dapps that allow devs to add new APIs to MetaMask, add new features, or modify existing functionality using internal APIs
- Snaps can only (at this time) be used in conjunction with MM Flask
- Read through the Snaps docs 
  - [Snaps Intro](https://docs.metamask.io/guide/snaps.html)
  - [JSON-RPC API](https://docs.metamask.io/guide/snaps-rpc-api.html)
  - [Dev Guide (under construction)](https://docs.metamask.io/guide/snaps-development-guide.html))
- Try out the [Sample Snap Tutorial](https://docs.metamask.io/guide/snaps.html#sample-snap-tutorial) (beginner level)
- Try out the [Sample FIlecoin Snap](https://github.com/ChainSafe/filsnap) (intermediate level)

Here are some additional resources: 

- [Flask and Snaps Filecoin Demo video](https://www.youtube.com/watch?v=14uEYdgyEr8)
- [MetaMask Extension Repo](https://github.com/MetaMask/metamask-extension)
- [Snaps Skunkworks Repo Examples](https://github.com/MetaMask/snaps-skunkworks/tree/main/packages/snap-examples/examples)

**Comment by ghost** [comment]
hey i used Vault Extraction to get my metamask back but it was not the only metamask i opened in that browser and i need it so much if someone can help all my money is there i lost my seed phrase please if someone can help !
